routes/socket/report.js

```diff
@@ -82,7 +82,7 @@ module.exports.makeReport = (data, game, type = 'report') => {
 					: `\n__**Message**__: ${httpEscapedSituation}\n__**Election #**__: ${election}\n__**Game Type**__: ${gameType}\n**<https://secrethitler.io/game/#/table/${uid}>**`
 			}`,
 			username: '@Mod Ping',
-			allowed_mentions: { parse: [] },
+			allowed_mentions: { parse: ['roles'] },
```

Comment:
```
const httpEscapedSituation = situation.replace(/( |^)(https?:\/\/\S+)( |$)/gm, '$1<$2>$3').replace(/@/g, '`@`');
```

(line 76) -- this does nothing because `@` is already escaped.